model/userModel.js

```diff
@@ -13,19 +13,31 @@ const userSchema = Mongoose.Schema({
     name: {
         type: String,
         required: [true, 'User name must be specified'],
-        trim: true
+        trim: true,
+        minlength: [1, 'User name must be at least 1 character long'],
+        maxlength: [255, 'User name cannot be longer than 255 characters']
     },
     surname: {
         type: String,
         required: [true, 'User surname must be specified'],
-        trim: true
+        trim: true,
+        minlength: [1, 'User surname must be at least 1 character long'],
+        maxlength: [255, 'User surname cannot be longer than 255 characters']
     },
     email: {
         type: String,
         required: [true, 'User email must be specified'],
-        match: [/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/, 'User email is not valid']
+        validate: {
+            validator: Validator.isEmail,
+            message: 'User email is not valid'
+        },
+        minlength: [1, 'User email must be at least 1 character long'],
```

Comment:
I don't think this is needed, because first of all you cannot create a valid email from one character, so the `Validator.isEmail` will fail. You can keep `maxlength` maybe, but `minlegth` is not needed.